src/plugins/js-eval/jsEvalPlugin.js

```diff
@@ -64,6 +64,7 @@ module.exports = async function jsEvalPlugin({ mentionUser, respond, message })
       '--experimental-vm-modules', // used by m>
       '--experimental-modules',
       '--no-warnings',
+      '-r', 'string.prototype.at/shim', 'array.prototype.at/shim',
```

Comment:
```suggestion
      '-r', 'string.prototype.at/auto', 'array.prototype.at/auto',
```

The shim entry point does nothing, it’s just a function.